aggregator/src/app/EthereumService.ts

```diff
@@ -302,6 +302,21 @@ export default class EthereumService {
 
       const attemptResult = await attempt();
 
+      try {
+        if (attemptResult.value.events[0]?.args?.results[0]) {
+          const result = attemptResult.value.events[0].args.results[0]; // For errors this is "Error(string)"
+          const errorArgBytesString = `0x${result.substring(10)}`; // remove methodId (4bytes after 0x)
+          const errorString = ethers.utils.defaultAbiCoder.decode(
+            ["string"],
+            errorArgBytesString,
+          )[0]; // decoded bytes is a string of the action index that errored.
```

Comment:
If https://github.com/web3well/bls-wallet/pull/366 goes in before this, you can just use that lib method.

Reply:
Yeah that is a good call.  I'll probably do that

Reply:
Actually available right meow https://github.com/web3well/bls-wallet/blob/main/contracts/clients/README.md#verificationgateway